helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -534,8 +535,34 @@ private Map<String, ResourceAssignment> partialRebalance(
       throw new HelixRebalanceException("Failed to generate cluster model for partial rebalance.",
           HelixRebalanceException.Type.INVALID_CLUSTER_STATUS, ex);
     }
-    Map<String, ResourceAssignment> newAssignment = calculateAssignment(clusterModel, algorithm);
 
+    _asyncPartialRebalanceResult = _bestPossibleCalculateExecutor.submit(() -> {
```

Comment:
would be useful to have log message about calculating Emergency pipeline. (i know method already has log statement, but ability to trace call-path will be useful)

Reply:
Could you elaborate on where logs should happen? This is the partial rebalance function.